net/goai/goai.go

```diff
@@ -65,6 +65,12 @@ const (
 const (
 	validationRuleKeyForRequired = `required`
 	validationRuleKeyForIn       = `in:`
+	validationRuleKeyMax         = `max`
```

Comment:
- Use `max:` instead of `max` for following string prefix check. And the same as `min:`, `length:`, `max-length:`, etc.
- Missing unit testing cases for this updtae.